### ISSUE TITLE
"The method 'toFilePath' was called on null." in FlutterWebPlatform

### ISSUE THREAD

**Issue by DanTup** [flutter/flutter#54971]
I saw this in the logs on DevTools CI a lot. It's not causing failures, but it looks like a bug:

```
GET /packages/ui/assets/Roboto-Regular.ttf
Error thrown by handler.
NoSuchMethodError: The method 'toFilePath' was called on null.
Receiver: null
Tried calling: toFilePath()
dart:core                                                        Object.noSuchMethod
package:flutter_tools/src/test/flutter_web_platform.dart 237:48  FlutterWebPlatform._packageFilesHandler
===== asynchronous gap ===========================
package:shelf/shelf_io.dart 63:34                                serveRequests.<fn>.<fn>
===== asynchronous gap ===========================
package:shelf/src/io_server.dart 53:5                            IOServer.mount
```

I don't know if the request to `packages/ui` is valid, but I think `_packageFilesHandler` should handle unresolvable packages gracefully (probably just with a 404).

@jonahwilliams

**Comment by jmagman** [comment]
\cc @mdebbar @jonahwilliams

**Comment by parnurzeal** [comment]
same issue here. I cannot run because of this error.
```
flutter test --platform chrome
```

**Comment by AresVampire** [comment]
I am facing this problem too.

**Comment by DanTup** [comment]
This was fixed in #59250.
